docs/faq/faq.md

```diff
@@ -62,6 +62,17 @@ pip3 install -U numexpr bottleneck matplotlib
 - If you start Autoware and `ros2 caret record` at the same time, operation may become unstable and some nodes may not be recorded.
 - It is recommended to start `ros2 caret record` a little while after starting Autoware.
 
+### How to measure a system with multiple ROS_DOMAIN_IDs
+
+CARET does not take "ROS_DOMAIN_ID" into account. Depending on the measurement method, it behaves as shown in the following table.
+
+| **No** | **The timing when the record command is executed** | **The domain where the record command is executed**                             | **Result**                                                          | **Note**                                                                                                      |
+| ------ | -------------------------------------------------- | ------------------------------------------------------------------------------- | ------------------------------------------------------------------- | ------------------------------------------------------------------------------------------------------------- |
+| 1      | Before the application starts                      | Execute the record command once for any domain (or without specifying a domain) | All domains are subject to measurement.                             | Namespace + Node name is OK if there are no duplicates.</br>Only domains with LD_PRELOAD set can be analyzed. |
```

Comment:
@xygyo77 
> Namespace + Node name is OK if there are no duplicates. 

“is OK” is vague and does not clearly indicate that the data can be analyzed.
Please clarify that analysis is possible only when the namespace/node_name pairs are unique.

As an alternative, how about the following sentence?

- The data can be analyzed as long as there are no duplicate namespace/node_name pairs.